specification/ai/Azure.AI.Projects/evaluations/models.tsp

```diff
@@ -176,3 +179,75 @@ model AgentEvaluation {
   @doc("The agent evaluation result.")
   result?: Array<AgentEvaluationResult>;
 }
+
+@doc("Abstract base model representing a single message in a conversation.")
+@discriminator("role")
+@added(Versions.v2025_05_15_preview)
+model Message {
+  @doc("The role of the message author: 'system', 'assistant', 'developer', or 'user'.")
+  role: string;
```

Comment:
If some strings have semantic meaning and/or it is a semi-closed set of values, we should model this as a `union`.

Reply:
Updated. Could you please review again?